Rubberduck.Parsing/VBA/ReferencePriorityMap.cs

```diff
@@ -8,16 +8,16 @@ namespace Rubberduck.Parsing.VBA
     /// </summary>
     public class ReferencePriorityMap : Dictionary<string, int>
     {
-        private readonly string _referenceId;
+        private readonly string _referencedProjectId;
 
-        public ReferencePriorityMap(string referenceId)
+        public ReferencePriorityMap(string referencedProjectId)
         {
-            _referenceId = referenceId;
+            _referencedProjectId = referencedProjectId;
         }
 
-        public string ReferenceId
+        public string ReferencedProjectId
         {
-            get { return _referenceId; }
+            get { return _referencedProjectId; }
```

Comment:
I hope so. The referenced projects have a combination of projectname+full path as their project id. I don't know if this holds up, would like to hear your thoughts on this. E.g. if you reference the MSXML library v3 and v6 it wouldn't work with just the project name, but with the full path it does since they use different dlls.